curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-string-and-array-methods-by-building-a-music-player/660ae8ad1475698da3c3c37d.md

```diff
@@ -29,8 +29,45 @@ Your `printGreeting` function call should be below the `printGreeting` function
 assert.isBelow(code.indexOf("const printGreeting = () => {"), code.indexOf("printGreeting()"));
 ```
 
+Your `printGreeting` function call should output `Hello there!` to the console. 
+
+```js
+assert.strictEqual(logOutput, "Hello there!"); 
+```
+
 # --seed--
 
+## --before-user-code--
+
+```js
+var logOutput = "";
```

Comment:
I get that this is "invisible" behind the scenes code, and that it gets passed through Babel, but why are we using `var`?